cli/cmd/root.go

```diff
@@ -53,3 +54,11 @@ func newPublicAPIClient() (pb.ApiClient, error) {
 	}
 	return public.NewExternalClient(controlPlaneNamespace, kubeApi)
 }
+
+// This is equivalent to calling log.Fatalf, but unlike go's logger interface,
+// it does not support any formatting of the output prior to printing, which is
+// preferable for printing errors from the CLI
+func logFatalf(format string, v ...interface{}) {
```

Comment:
I actually like the formatted output, but if we're going to change this then I'd prefer to have a global `log` object that is configured this way on `root`. Whatever solution we end up with, I think the important thing is that either we _only_ use the wrapper functions or we _only_ use the log object directly, which means that we'd need wrapped functions for all levels and formatted vs. `ln` versions.

Reply:
Cool, yeah this all makes sense. I've removed the `logFatalf` method, and changed the initializer back to setting the log level on logrus's global logger. Related to my comment above, I've also defaulted the log level to "panic" (no logging), when the `--verbose` flag is not set. As part of this change, I've converted all of the `log.Fatal` calls in this package to instead print directly to stderr (since they otherwise wouldn't be printed at the default level). This guarantees that we won't print log messages unless the `--verbose` flag is set, and when we do print messages, they'll be fully formatted with timestamps.